src/data/Data.test.ts

```diff
@@ -341,7 +371,8 @@ describe('WorkspaceData', () => {
 
     // Assert
     expect(mockListAppsV2).toHaveBeenCalledTimes(3); // third call, now running
-    expect(mockGetSchema).toHaveBeenCalled(); // fetch schema once running
+    expect(mockListAppsV2).toHaveBeenCalledWith('test-workspace-id'); // it should have been called with the correct ID
```

Comment:
Adding a comment to cross-ref from another PR to demonstrate the importance of verifying for specific args vs. verifying 'any' or loose args.